pixi/src/ui/report/CoreWebVitalsReport.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export default class CoreWebVitalsReport {
+  constructor(reportData) {
```

Comment:
Suggestion: separate initialization and rendering

```
const coreWebVitalsReport = new CoreWebVitalsReport(document)
coreWebVitalsReport.render(report.coreWebVitals)
```

Sidenote: passing document via constructor makes it easier to implement tests.